packages/cli/lang/en.lyaml

```diff
@@ -1077,7 +1077,7 @@ en:
             errors:
               invalidName: "You entered an invalid name. Please try again."
               nameRequired: "The name may not be blank. Please try again."
-              accountNameExists: "Account name \"{{ name }}\" already exists, please enter a different name."
+              accountNameExists: "Account with name \"{{ name }}\" already exists in the CLI config, please enter a different name."
```

Comment:
Adjusted in two places to make it clearer what the "name" error is referencing

Reply:
Nice, this does make it much more clear 👍